test/infrastructure/docker/controllers/dockermachine_controller_test.go

```diff
@@ -71,23 +75,8 @@ func TestDockerMachineReconciler_DockerClusterToDockerMachines(t *testing.T) {
 	for i := range out {
 		machineNames[i] = out[i].Name
 	}
-	if len(out) != 2 {
-		t.Fatal("expected 2 docker machines to reconcile but got", len(out))
-	}
-	for _, expectedName := range []string{"my-machine-0", "my-machine-1"} {
-		if !contains(machineNames, expectedName) {
-			t.Fatalf("expected %q in slice %v", expectedName, machineNames)
-		}
-	}
-}
-
-func contains(haystack []string, needle string) bool {
-	for _, straw := range haystack {
-		if straw == needle {
-			return true
-		}
-	}
-	return false
+	g.Expect(out).To(HaveLen(2))
+	g.Expect(machineNames).To(ContainElements("my-machine-0", "my-machine-1"))
```

Comment:
I think this can be a `ConsistOf`? The `machineNames` are constructed from `out` which we require to have length 2, so the elements listed here should be the entire list of names. I think technically you could skip the `g.Expect(out).To(HaveLen(2))` if you used `ConsistOf` since it will check there are no extra elements, but I think it's fine to keep for now so it's more explicit
```suggestion
	g.Expect(machineNames).To(ConsistOf("my-machine-0", "my-machine-1"))
```

Reply:
made the change, thanks for your review!